### PR TITLE
Add commas

### PR DESCRIPTION
**Pull request by jeph864**

### CODE REVIEW

File: plugin.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 $id = "xbbb";
  
 // code version; must be changed for all code changes
-$version = "1.0.14";
```
Comment (Rillke) [inline plugin.php:7]: The commit message and pull request title does not match what's done in the code.

You are actually increasing the version number.

Comment (Rillke) [reply plugin.php:7]: And actually, it's already merged, so this PR can be closed.

**Review by Rillke — CHANGES_REQUESTED** [review]
Please close this PR.

**Comment by jeph864** [comment]
@Rillke 
thanks for the info!

**Pull request closed without merging**